public/main.js

```diff
@@ -44,6 +43,7 @@ const getPosts = () => {
           }
         };
 
+        const lines = contents.split("\n");
```

Comment:
I wonder how this works - const variables are not hoisted, right? so how does parseMetadata and parseContent functions access this variable if lines were declared later?

Reply:
What matters is when the functions are invoked. Of course, they need to be defined before invocation.